### PR TITLE
Upgrade Kinto/Cliquet bugfixes releases

### PR DESCRIPTION
**Pull request by leplatrem**
@Natim r?

### CODE REVIEW

**Comment by Natim** [comment]
If it is a bugfix we probably don't need to upgrade kinto-dist because the previous version will take the last bugfix release. Do you have a specific case in mind?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,7 +7,10 @@ the version control of each dependency.
 0.6.0 (unreleased)
```
Comment (Natim) [inline CHANGELOG.rst]: Shouldn't it be 0.5.1?

Comment (leplatrem) [inline CHANGELOG.rst]: Ok yes, we can take advantage of this PR to prepare a release.

**Comment by leplatrem** [comment]
> Do you have a specific case in mind?

Oh, not really. More like an automatism...

**Comment by Natim** [comment]
Let's do it then :)

**Pull request merged**